static/css/jquery.tooltips.css

```diff
@@ -1,9 +1,13 @@
+.tooltip strong, b {
```

Comment:
I think this should be `.tooltip strong, .tooltip b {`, but it's not having any effect for me.  I don't think we have an 900 weight font.

I tried changing https://github.com/nightscout/cgm-remote-monitor/blob/52e1ff054c98a6c0ee90d6f3b4d5c299e2b6dd40/static/css/main.css#L2 to 
`@import url('https://fonts.googleapis.com/css?family=Ubuntu:400,700');`, based on https://fonts.google.com/specimen/Ubuntu?selection.family=Ubuntu:400,700 using `font-weight: 700`, thats the heavyist they seem to have for the font.  In the old version that I'm still running we were using `font-family: 'Open Sans', Helvetica, Arial, sans-serif;` on the body element.  I think @sulkaharo did some [optimizations](https://github.com/nightscout/cgm-remote-monitor/commit/936d631870132ac931daaa75d04aa46961bcc5c7), but now it's https://github.com/nightscout/cgm-remote-monitor/blob/52e1ff054c98a6c0ee90d6f3b4d5c299e2b6dd40/static/css/main.css#L74

Reply:
you're right. fixed and merged your PR in. I don't know if we use `b` within `tooltip`s